packages/yoroi-extension/features/memo.feature

```diff
@@ -37,6 +37,6 @@ Feature: Memos
       | password   |
       | asdfasdfasdf |
     And I submit the wallet send form
+    Then I should see the successfully sent page
+    And I click the transaction page button
     Then I should see the summary screen
-    And I expand the top transaction
-    Then The memo content says "my awesome memo"
```

Comment:
Please do not delete these two lines. Because the purpose of this test is to check that these lines are true.

Reply:
Okay, I will get them back. And run the test again